src/main/java/org/mifos/pheeBillPay/zeebe/ZeebeWorkers.java

```diff
@@ -157,6 +160,18 @@ public void setupWorkers() {
             String body = variables.get(BILL_RTP_REQ).toString();
             String billerId  = variables.get("billerId").toString();
             BillRTPReqDTO billRTPReqDTO = objectMapper.readValue(body, BillRTPReqDTO.class);
+            if(billRTPReqDTO.getPayerFspDetail() != null) {
+                if (billRTPReqDTO.getPayerFspDetail().getPayerFspId().equals("rhino") && billRTPReqDTO.getPayerFspDetail().getFinancialAddress().equals("122333")) {
```

Comment:
please donot hardcode these values, read them from application.yml.
Tenants should always be configurable in Paymenthib

Reply:
Added these values in application.yaml